posts/2022-10-27-22.0.0.12-beta.adoc

```diff
@@ -0,0 +1,356 @@
+[source,xml]
+---- 
+<featureManager>
+   <feature>messaging-3.1</feature>
+</featureManager>   
+----
+
+For more information about Jakarta Messaging, refer to the following links:  
+
+* link:{url-prefix}/docs/latest/reference/feature/messaging.html[Feature Documentation: Jakarta Messaging]
+* link:https://jakarta.ee/specifications/messaging/3.1/jakarta-messaging-spec-3.1.html[Specification: Jakarta XML Web Services]
+
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/22902
+// Contact/Reviewer: neuwerk,ReeceNana
+// // // // // // // // 
+[#xmlws]
+== Jakarta Web Services Metadata incorporated into XML Web Services 4.0
+
+Jakarta XML Web Services 4.0 is a set of APIs that allows applications to deploy SOAP Based Web Services on the Jakarta EE 10 platform. This set of APIs includes both the XML Web Services 4.0 specification which now incorporates the Jakarta Web Services Metadata specification, as well as the Jakarta SOAP with Attachments 3.0 specification. 
+
+Although the changes introduced in XML Web Services 4.0 are relatively minor, they do include the requirement for Java SE 11 as well as incorporating the Jakarta Web Services Metadata specification into XML Web Services 4.0. 
+
+
+To enable the new Jakarta XML Web Services 4.0 feature, add the `xmlWS-4.0` feature to your server.xml:
+[source,xml]
+---- 
+<featureManager>
+   <feature>xmlWS-4.0</feature>
+</featureManager>   
+----
+
+For more information about Jakarta XML Web Services, refer to the following links:  
+
+* link:{url-prefix}/docs/latest/reference/feature/xmlWS.html[Feature Documentation: Jakarta XML Web Services]
+* link:https://jakarta.ee/specifications/xml-web-services/4.0/jakarta-xml-ws-spec-4.0.html[Specification: Jakarta XML Web Services]
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/22930
+// Contact/Reviewer: teddyjtorres,ReeceNana
+// // // // // // // // 
+[#jwt]
+== Simplified JWT Validation (MicroProfile JSON Web Token 2.1)
+The MicroProfile JSON Web Token 2.1 specification allows the use of a JWT token for authenticating and authorizing requests to a service. The specification simplifies the configuration for managing the validation of the JWT by introducing three new MicroProfile Config properties along with corresponding `mpJwt` attributes.
+
+=== mp.jwt.verify.token.age
+The `mp.jwt.verify.token.age` property allows you to specify the number of **seconds** since the JWT token was issued. The "iat" claim must be present in the JWT and the configured number of seconds since "iat" must not have elapsed. If it has elapsed, then the request is rejected with an Unauthorized (401) response.

```

Comment:
```suggestion
The `mp.jwt.verify.token.age` property allows you to specify the number of **seconds** since the JWT token was issued. The `iat` (issued at) claim must be present in the JWT and the configured number of seconds since `iat` must not have elapsed. If it has elapsed, then the request is rejected with an Unauthorized (401) response.

```